### PR TITLE
Add panic dump debug support for renoir

### PR DESCRIPTION
**Pull request by balakishorepati**
This pull request enables the support to dump panic info during any firmware exceptions on renoir platform.
Fix for IPC timeouts during firmware boot up time.
Add code owners for AMD platforms.

### CODE REVIEW

File: src/platform/amd/renoir/include/platform/drivers/interrupt.h [hunk 1]
```diff
@@ -26,13 +26,13 @@
...
+#define IRQ_NUM_EXT_LEVEL3	3	/* level 3 */
 
-#define IRQ_NUM_TIMER1		6	/* level 2 */
+#define IRQ_NUM_TIMER1		6	/* level 5 */
 
-#define IRQ_NUM_EXT_LEVEL4	4	/* level 2 */
+#define IRQ_NUM_EXT_LEVEL4	4	/* level 4 */
 
-#define IRQ_NUM_EXT_LEVEL5	5	/* level 3 */
+#define IRQ_NUM_EXT_LEVEL5	5	/* level 5 */
```
Comment (dbaluta) [inline src/platform/amd/renoir/include/platform/drivers/interrupt.h:36]: This is just fixing some comments and needs to be added in a separate patch. Remember one patch per logical unit change.

Comment (balakishorepati) [reply src/platform/amd/renoir/include/platform/drivers/interrupt.h]: Removed the comment change from the patch.

**Review by lgirdwood — APPROVED** [review]

**Comment by lgirdwood** [comment]
Looks like CI needs rerun as this PR is unrelated and should have no impact.

**Comment by lgirdwood** [comment]
SOFCI TEST

**Comment by lgirdwood** [comment]
@balakishorepati btw, please feel free to add Renoir to the platform list on the sof-docs repo. It's used to generate https://thesofproject.github.io/latest/platforms/index.html

**Review by cujomalainey — APPROVED** [review]

**Comment by lgirdwood** [comment]
CI showing unrelated issues. retest.

**Comment by lgirdwood** [comment]
SOFCI TEST

**Comment by lgirdwood** [comment]
CI CML issues are unrelated and known.
